avtozip/store/templates/store/index.html

```diff
@@ -1,8 +1,21 @@
 {% extends "store/main.html" %}
 
+{% load staticfiles %}
+
 {% block title %}AvtoZip Store{% endblock %}
 
 {% block content %}
+  <link rel="stylesheet" type="text/css" href="{% static 'store/css/main.css' %}" />
```

Comment:
Create css section in general template in head
Then redefine it here and place this css there